### DISCUSSION TITLE
Prestissimo TPCDS performance

### DISCUSSION THREAD

**Discussion by sumokiama** [prestodb/presto#23775] [General]
We have setup an AWS 2xr5dn.8xlarge Prestissimo cluster and have set up the hive connector to access our S3 TPCDS datasets. I works but performance is pretty slow...about the same speed as stock Spark on EMR.

Is there a tuning guide that for configuring Velox memory, threading and asynchronous caching (memory and SSD) based on Prestissimo properties that would optimize TPCDS performance?

We've seen the TPCH tuning guide but that does not help for TPCDS.

**Comment by majetideepak** [comment]
@sumokiama what is the TPCDS scale factor used for the evaluation?
